foolbox/batching.py

```diff
@@ -41,10 +41,10 @@ def run_sequential(create_attack_fn, model, criterion, inputs, labels,
         if the threshold has been reached.
     verbose : bool
         Whether the adversarial examples should be created in verbose mode.
-    attack_kwargs : dict or list of dict
-         The optional keywords passed to create_attack_fn. If a dict, the same
-         values will be used for all inputs; if a list, for each input a
-         different set of arguments will be used.
+    individual_kwargs : list of dict
+         The optional keywords passed to create_attack_fn that should be
+         different for each of the input samples. For each input a different
+         set of arguments will be used.
```

Comment:
I think we should still document kwargs here, because it's still supported as a way to specify shared kwargs for all attacks …

Reply:
You're right - I've added it to the docstring again.